apps/web/src/api/v2/util.ts

```diff
@@ -56,7 +56,8 @@ export const createBuild = async ({
   if (!isPublic) {
     const account = await req.authProject.$relatedQuery("account", trx);
     const hasExceedLimit = await account.hasExceedScreenshotsMonthlyLimit();
-    if (hasExceedLimit) {
+    const hasUsageBasedPlan = await account.hasUsageBasedPlan();
```

Comment:
I think methods are too abstract. Then you fetch the same thing multiple times. Not a big deal but maybe to refactor later.